src/pm_wrapper.jl

```diff
@@ -1,9 +1,85 @@
-run_opf(system::PSY.System, )
-run_pf(system::PSY.System, )
-compute_ac_pf(system::PSY.System, )
-run_ac_pf(system::PSY.System, )
-compute_dc_pf(system::PSY.System, )
-run_dc_pf(system::PSY.System, )
-
-run_model(system::PSY.System,)
-instantiate_model(system::PSY.System,)
+
+function run_opf(system::PSY.System, model_type::Type, optimizer; kwargs...)
+    pmi_data = get_pm_data(system; kwargs...)
+    return run_opf(
+        pmi_data,
+        model_type,
+        optimizer;
+        (k for k in kwargs if first(k) ∉ ACCEPTED_PM_DATA_KWARGS)...,
+    )
+end
+
+function run_mn_opf(system::PSY.System, model_type::Type, optimizer; kwargs...)
+    pmi_data = get_pm_data(system; kwargs...)
+    return run_mn_opf(
+        pmi_data,
+        model_type,
+        optimizer;
+        (k for k in kwargs if first(k) ∉ ACCEPTED_PM_DATA_KWARGS)...,
+    )
+end
```

Comment:
we should merge these two functions and just identify when the user passes a time_periods kwarg.